meta/def.rb

```diff
@@ -94,6 +110,7 @@
   mutation 'def foo(_unused = false); end'
   mutation 'def foo(_unused = true); raise; end'
   mutation 'def foo(_unused); end'
+  mutation 'def foo; _unused = true; end'
```

Comment:
Ah I'm confused then.  #310 certainly specifies `def foo(arg = default); end` → `def foo; arg = default`. Shall we just go with `def foo(arg = default); arg = default; end` then as the mutation?  that seems like it would be easier actually